_config.yml

```diff
@@ -5,8 +5,8 @@
 # 'bundle exec jekyll serve'. If you change this file, please restart the server process.
 
 # Site title and description
-title:              OAK Docs
-description:        Documentation for OAK Blockchain
+title:              Ava Protocol Docs
+description:        Documentation for Ava Protocol Blockchain
```

Comment:
Remove the word "Blockchain", "Ava Protocol" doesn’t go with Blockchain

Reply:
Fixed